HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,124 @@
-data("iris")
-```
-
-Using the iris dataset, make this graph:
-```{r graph 2 code, echo=FALSE}
-
-```
-
-
-### Graph 3
-You'll need the information in this first box to create the graph
-```{r, echo = TRUE}
-data("mpg")
-corvette <- mpg[mpg$model == "corvette",]
-#install
-require("ggrepel") #useful for making text annotations better, hint hint
-set.seed(42)
-```
-
-Now using the mpg dataset and the corvette dataset, make this graph:
-
-```{r graoh 3 code}
-
-```
-
-There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
-
-
-### Graph 4 
-```{r, echo = T}
-data(mpg)
-
-#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
-library(RColorBrewer)
-display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
-```
-
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
-
-
-
-Now using the above mpg dataset, make this graph
-
-```{r graph 4 code}
-
-```
-
-
-
-
+---
+title: "HW02_B_Graph-Mimic"
+author: "Kelly O'Leary"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+ggplot(diamonds, aes(x = cut), group = cut) +
+ geom_bar(aes(fill = clarity), position = position_dodge()) +
+ theme(legend.position = "right") +
+ labs(title="My Diamond Collection", 
+       subtitle="Boxplot representing the number of diamonds in my diamond collection by type of cut quality and clarity of diamond",
+       caption="My Best Diamonds, of course",
+       x="Diamond Cut",
+       y="Number of Diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+ggplot(iris, aes(x = Sepal.Length, y = Petal.Length, color = Species)) +
+ geom_point(data = subset(iris, Species %in% c("setosa", "virginica", "versicolor"))) +
+  facet_wrap(~Species) +
```

Comment:
Congrats on finding a way to group the data into panels. Honestly I spent most of my time on this graph. In order to re-order the panels I found a way to generate a new subset with re-ordered data (iris$Species2 <- factor(iris$Species, levels = c("versicolor", "setosa", "virginica")). and then passed this subset to facet_wrap(). And to change scales range I used scales = free_y. Anyway, GJ! That was a tricky one.